hello_world_on_chain/contracts/User.sol

```diff
@@ -19,18 +28,23 @@ contract User {
             applicationPublicKey
         );
 
+        uint256 startGas = gasleft();
+
         claims = TAParserLib.verifyTransitivelyAttestedFnCall(
-                applicationPublicKeyAsAddress,
-                transitiveAttestation
-            );
+            applicationPublicKeyAsAddress,
+            transitiveAttestation
+        );
+
+        uint256 endGas = gasleft();
+        console.log("Gas used: %s", startGas-endGas);
```

Comment:
Note that this produces the output:
```
Gas used: 14423                                                                                                                                                                                                                             ```

Reply:
Sick!

Reply:
We should probably take this out to the output though.